packages/agoric-cli/src/lib/casting.js

```diff
@@ -0,0 +1,38 @@
+// @ts-check
+
+import { delay, exponentialBackoff, randomBackoff } from '@agoric/casting';
```

Comment:
I suspect `delay` closes over `setTimeout`, so it's powerful. sigh.

Reply:
noted